packages/rum/src/domain/rumEventsCollection/view/trackTimings.ts

```diff
@@ -57,3 +65,42 @@ export function trackFirstContentfulPaint(lifeCycle: LifeCycle, callback: (fcp:
   })
   return { stop }
 }
+
+export function trackLargestContentfulPaint(
+  lifeCycle: LifeCycle,
+  emitter: EventEmitter,
+  callback: (value: number) => void
+) {
+  const firstHidden = trackFirstHidden()
+
+  // Ignore entries that come after the first user interaction
```

Comment:
I can't find a link to show that it is needed. All the documentation (https://web.dev/lcp/#when-is-largest-contentful-paint-reported https://github.com/WICG/largest-contentful-paint/blob/master/README.md#interaction-with-user-input  https://wicg.github.io/largest-contentful-paint/#sec-add-lcp-entry ) seems to agree that the browser won't send those performance entries after the first user interaction. I think that's just a safeguard that web-vitals implemented because of some Chrome issues.